README.md

```diff
@@ -26,19 +28,24 @@
 
 ## Get started
 
-Flagd is a simple command line tool for fetching and evaluating feature flags for services. It is designed to conform with the OpenFeature specification. To get started, follow the installation instructions in the [docs](https://github.com/open-feature/flagd/blob/main/docs/README.md).
+Flagd is a simple command line tool for fetching and evaluating feature flags for services.
+It is designed to conform with the OpenFeature specification.
+To get started, follow the installation instructions in the [docs](https://github.com/open-feature/flagd/blob/main/docs/README.md).
```

Comment:
Can you make this a relative path?

Reply:
I would prefer to do this in a next clean up. eg. the configuration/configuration.md should be also renamed to configuration/README.md 

i see this than more as general improvements to the documentation and structure - rather than this simple cleaning/adapting to linting